plutus-conformance/add-test-output/Spec.hs

```diff
@@ -82,20 +83,25 @@ allOrMissing = missing <|> allInputs
 
 args :: ParserInfo Args
 args = info ((MkArgs <$> ext <*> dir <*> runner <*> allOrMissing) <**> helper)
-  (fullDesc <> progDesc helpText)
+  (fullDesc <> progDescDoc (Just helpText))
```

Comment:
Leave here a comment about `progDesc` butchering formatting, so that nobody is tempted to simplify `progDescDoc` to it.